src/Components/CatProfile.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useState, useEffect, ChangeEvent } from 'react';
+import { getCatsProfiles } from '../api';
+
+function CatProfile() {
+	const [changeName, setChangeName] = useState<string>('');
+	const [changePictureurl, setChangePictureurl] = useState<string>('');
+	const [changeDescription, setChangeDescription] = useState<string>('');
```

Comment:
I'm confused by the naming. Should these not just be 'name', 'pictureUrl', and 'description'?